### ISSUE TITLE
[BUG] MAPINFO errors

### ISSUE THREAD

**Issue by lemonzest79** [odamex/odamex#358]
Hi I'm using Odamex 0.9.0 Windows Build and some wads will not load due to MAPINFO errors, I have quite a few that do not launch

Doom II
haxc.wad HacX 1.2 IWAD MAPINFO:9:Script Error: Unknown MAPINFO token "M_TWITCH"
cafebrk1.wad MAPINFO:4:Script Error:Unknown MAPINFO token "compat_nopassover"
epic2.wad ZMAPINFO:14:Script Error:Expected = got "nojump"
jadearth.wad/deh MAPINFO:5:Script Error:Bad floating point constant "nocrouch"
mayhem18o/p.wad MAPINFO:8:Script Error:Bad floating point constant "next"
mayhem19.wad MAPINFO:8:Script Error:Bad floating point constant "next"
mayhem20.wad MAPINFO:13:Script Error:Unknown lookup string "Wet My Appetite"
mayhem2048.wad MAPINFO:318:Script error:Unknown MAPINFO token "Entertext"YOU"
pc_cp.wad MAPINFO:9:Script error:Unknown MAPINFO token "DOOM 2 UNLEASHED"

Ultimate Doom
lostepis.wad MAPINFO:6:Script Error:Unknown MAPINFO token "Evil Unleashed"

**Comment by Ch0wW** [comment]
I've checked the list of wads, and I've been able to fix the following on my PR. Here's the list with its upcoming state:

```
- hacx.wad : now works
- cafebrk1.wad : now works
- epic2.wad : still not fixed (newmapinfo parser error?)
- jadearth.wad : now works
- MAYhem18O/P.wad : now works
- MAYhem19.wad : now works
- mayhem20.wad : now works
- mayhem2048.wad : still unfixed (believes argument is entertext"you )
- pc_cp.wad : now works
- lostepis.wad : now works
```

**Comment by rakohus** [comment]
Some things have changed between @Ch0wW's comment and @DeathEgg's new MAPINFO branch. I tested this list and here's the current situation:

```
- hacx.wad : works
- cafebrk1.wad : works
- epic2.wad : ZMAPINFO parsing error
- jadearth.wad : MAPINFO parsing error
- MAYhem18O/P.wad : MAPINFO parsing error
- MAYhem19.wad : MAPINFO parsing error
- mayhem20.wad : MAPINFO parsing error
- mayhem2048.wad : works
- pc_cp.wad : works
- lostepis.wad : MAPINFO parsing error

**Comment by DeathEgg** [comment]
> Some things have changed between @Ch0wW's comment and @DeathEgg's new MAPINFO branch. I tested this list and here's the current situation:
> 
> ```
> - hacx.wad : works
> - cafebrk1.wad : works
> - epic2.wad : ZMAPINFO parsing error
> - jadearth.wad : MAPINFO parsing error
> - MAYhem18O/P.wad : MAPINFO parsing error
> - MAYhem19.wad : MAPINFO parsing error
> - mayhem20.wad : MAPINFO parsing error
> - mayhem2048.wad : works
> - pc_cp.wad : works
> - lostepis.wad : MAPINFO parsing error
> ```

All of these should be fixed now as of PR 544. `lostepis.wad` still isn't loading but for seemingly unrelated reasons.

**Comment by rakohus** [comment]
Confirming all but `lostepis.wad` fixed with #600. `lostepis.wad` is a doom pwad, so load with doom.wad. 

Relevant error: 
![image](https://user-images.githubusercontent.com/22926877/156978747-23aac65c-0a5c-4065-b135-fcab03fcd13a.png)

**Comment by rakohus** [comment]
All wads listed above now load in stable after #615. Thanks @DeathEgg
